src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Modal.scss';
+
+const Modal = (props) => {
```

Comment:
Can we please destructure the props. Also, make sure the to install the new dependencies using `npm install` and install the `ESLint` plugin from this [link](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint);

Reply:
Also, if required take a pull from the `master` so that you have the latest code. Rebase your branch with the `master` and then push the code.

Reply:
@madhuni Thanks for the insights, I have implemented your suggestions. Regarding ES lint, when I start the project after installing the new dependencies I get an error that CRA uses `"eslint": "^6.6.0"` and the project uses `7.4.0`. Did you see this?
![Screen Shot 2020-07-08 at 4 34 02 PM](https://user-images.githubusercontent.com/54119818/86925061-020e2700-c139-11ea-8108-f7227b57c8e6.png)
 I deleted the node_modules and package-lock.json but it didn't help. Thanks for the help.

Reply:
Okie let me have a look. Will provide you the solution to resolve this. I didn't see any such problem.